tests/common/devices/multi_asic.py

```diff
@@ -94,6 +96,9 @@ def critical_services_tracking_list(self):
                     self.sonichost.DEFAULT_ASIC_SERVICES.remove(service)
                 if config_facts['FEATURE'][service]['state'] == "disabled":
                     self.sonichost.DEFAULT_ASIC_SERVICES.remove(service)
+        else:
+            self._DEFAULT_SERVICES.append("lldp")
```

Comment:
do we need this append here?
This will append lldp DEFAULT_SERVICES for linecard module as well.
Just tried this on a Linecard, and I see that this function https://github.com/sonic-net/sonic-mgmt/blob/67a8074b0bd63c456c315e2649208fcdb4f89045/tests/common/devices/sonic.py#L250C5-L250C35 returns "True"
>>> import sonic_platform
>>> import sonic_platform.platform as P
>>> print(P.Platform().get_chassis().is_modular_chassis())
True

Reply:
@SuvarnaMeenakshi 👍 

Yes .  LLDP will be added as Default Service for Supervisor and Fixed Platoforms (else is for Fixed platforms)

For Modular chassis LLDP will not be added to Default Service List which means for LC we will not add as per our requirement.

Reply:
Am I missing something?

Reply:
got it, misunderstood the condition logic here.